OpenRA.Mods.Cnc/Traits/Infiltration/Infiltrates.cs

```diff
@@ -119,7 +119,7 @@ public void ResolveOrder(Actor self, Order order)
 			if (!order.Queued)
 				self.CancelActivity();
 
-			self.SetTargetLine(target, Color.Red);
+			self.ShowTargetLines();
```

Comment:
Unless SetTargetLine's display parameter was explicitly specified as false, the game did show the target line. In Infiltrates.cs' case, this is correct. In some cases I put ShowTargetLines is false. I'll fix that